terraform/shared/shared.tf.json.template.py

```diff
@@ -909,14 +906,45 @@ def conformance_pack(name: str) -> str:
             **{
                 f'best_practices_macie_{num}': {
                     'standards_control_arn': f'arn:aws:securityhub:{aws.region_name}:{aws.account}:control'
-                                             f'/aws-foundational-security-best-practices/v/1.0.0/Macie.{num}',
+                                             f'/nist-800-53/v/5.0.0/Macie.{num}',
                     'control_status': 'DISABLED',
                     'disabled_reason': 'Generates alarm noise; tracked independently as follow-up work',
                     'depends_on': [
-                        'aws_securityhub_standards_subscription.best_practices'
+                        'aws_securityhub_standards_subscription.nist_800_53'
                     ]
                 }
                 for num in [1, 2]
+            },
+            **{
+                f'nist_non_moderate_control_{i}': {
+                    'standards_control_arn': f'arn:aws:securityhub:{aws.region_name}:{aws.account}:control'
+                                             f'/nist-800-53/v/5.0.0/{control}',
+                    'control_status': 'DISABLED',
+                    'disabled_reason': 'Not a moderate level control',
+                    'depends_on': [
+                        'aws_securityhub_standards_subscription.nist_800_53'
+                    ]
+                }
+                for i, control in enumerate([
+                    'ACM.1',
+                    'CloudFront.1',
+                    'S3.15',
+                    #
+                    # EFS.6 is excluded since despite it being listed as a
+                    # control that applies to NIST SP 800-53 Rev. 5 …
+                    # https://docs.aws.amazon.com/securityhub/latest/userguide/nist-standard.html
+                    # … it is not. Other AWS documentation backs up this claim.
+                    # https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-controls-reference.html
+                    # 'EFS.6',
```

Comment:
```suggestion
                    # EFS.6 is excluded since despite it being listed as a
                    # control applicable to NIST SP 800-53 Rev. 5 …
                    #
                    # https://docs.aws.amazon.com/securityhub/latest/userguide/nist-standard.html
                    #
                    # … but it is not. Other AWS documentation backs up this claim:
                    #
                    # https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-controls-reference.html
                    # 'EFS.6',
```